syntax = "proto3";

import "streamlit/proto/LabelVisibilityMessage.proto";

message DateInput {
  string id = 1;
  string label = 2;
  repeated string default = 3;
  string min = 4;
  string max = 5;
  bool is_range = 6;
  string help = 7;
  string form_id = 8;
  repeated string value = 9;
  bool set_value = 10;
  bool disabled = 11;
  LabelVisibilityMessage label_visibility = 12;
}
